applications/portfolioDrafts/portfolio/createPortfolioStep.ts

```diff
@@ -20,9 +21,28 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
     );
   }
 
-  const portfolioId = event.pathParameters?.portfolioDraftId;
-
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+  if (!portfolioDraftId) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "PortfolioDraftId must be specified in the URL path" },
+      ErrorStatusCode.BAD_REQUEST
+    );
+  }
+  if (!IsValidJson(event.body)) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "Invalid request body: Invalid JSON" },
+      ErrorStatusCode.BAD_REQUEST
+    );
+  }
   const requestBody = JSON.parse(event.body);
+
+  if (!isPortfolioStep(requestBody)) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "Invalid request body: Missing request attributes" },
```

Comment:
Per the specification, you need to return an `error_map`. But maybe we handle this in a future PR with proper validation